### ISSUE TITLE
Error with karma-tap@4.1.1 (adapter that implements __karma__.start method)

### ISSUE THREAD

**Issue by thisconnect** [bySabi/karma-tap#29]
Hi!

my simple test passes with `karma-tap@4.1.0`
https://github.com/thisconnect/test-karma-rollup-tape/commits/master

but I get the following error when updating from `karma-tap@4.1.0` to `karma-tap@4.1.1`

```bash
> karma start karma.conf.js

Chrome 63.0.3239 (Mac OS X 10.13.3) ERROR
  {
    "message": "You need to include some adapter that implements __karma__.start method!",
    "str": "You need to include some adapter that implements __karma__.start method!"
  }


Firefox 58.0.0 (Mac OS X 10.13.0) ERROR
  {
    "message": "You need to include some adapter that implements __karma__.start method!",
    "str": "You need to include some adapter that implements __karma__.start method!"
  }
```

all I could find was this long thread, but no pointer https://stackoverflow.com/questions/24220888/error-you-need-to-include-some-adapter-that-implements-karma-start-method

related  https://github.com/thisconnect/test-karma-rollup-tape/issues/39

**Comment by bySabi** [comment]
Solved on release `4.1.3` and `3.2.1` 
Please update.

**Comment by thisconnect** [comment]
@bySabi thank you!
